Vizzuality/waterbird-population
# Fix/icons,faqs

This PR:
- Fixes changes the icons back to how they were.
- Fixes bug when trying to go to FAQ page.
- Add background image to static pages.

## Testing instructions

- All icons in page should appear correctly apart from "download icon".
- Go to http://localhost:3000/faq
